### PR TITLE
[test] Cross reference chip_sw_rstmgr_alert_info test

### PR DESCRIPTION
**Pull request by msfschaffner**
This fixes https://github.com/lowRISC/opentitan/issues/14140.

### CODE REVIEW

**Comment by msfschaffner** [comment]
@jdonjdon, the issue mentions you are already assigned to that testpoint. Please let me know if this collides with work you have already done.

File: hw/top_earlgrey/data/chip_testplan.hjson
Comment (tjaychen) [inline hw/top_earlgrey/data/chip_testplan.hjson:1343]: i actually double checked this with @engdoreis some time back.
I think they felt that there were enhancements needed. I'm not sure if they were planning to write a new test, or enhance the existing one.

Comment (jdonjdon) [reply hw/top_earlgrey/data/chip_testplan.hjson:1343]: Oh.. I updated it here(https://github.com/lowRISC/opentitan/pull/14894)
still pending but I added local alert after reviewing with @cindychip

Comment (msfschaffner) [reply hw/top_earlgrey/data/chip_testplan.hjson:1343]: Ah great, thanks! Looks like that just updates the `chip_sw_rstmgr_alert_info` test, right?

Should we close this PR here, or just wait until your update is merged?

Comment (jdonjdon) [reply hw/top_earlgrey/data/chip_testplan.hjson:1343]: @msfschaffner  Let's keep this open until #14894  is merged 
if there is no urgency.

Comment (jdonjdon) [reply hw/top_earlgrey/data/chip_testplan.hjson:1343]: @msfschaffner #14894 is merge. You can close this whenever you can.

**Comment by msfschaffner** [comment]
Can someone approve the PR :)?

**Review by tjaychen — APPROVED** [review]

**Comment by msfschaffner** [comment]
Thanks!

**Pull request merged**
